src/chrome/content/rules/Connect.me.xml

```diff
@@ -1,22 +1,16 @@
-<!--
-	^connect.me: Expired, missing certificate chain
-	www.connect.me: Refused
-
--->
 <ruleset name="Connect.me">
 
 	<!--	Complications:
 				-->
-  <target host="connect.me"/>
```

Comment:
`connect.me` and `www.connect.me` both point to different hosts. At the moment, `connect.me` is undergoing some changes, so the https redirects weren't happening correctly. My intention was to get `www.connect.me` fixed properly and then, when `connect.me` became stable to submit another PR to update that URL.